### DISCUSSION TITLE
Cell Output Width running R kernel

### DISCUSSION THREAD

**Discussion by pehkawn** [microsoft/vscode-jupyter#9747] [Questions and Answers] [answered]
This issue relates to issue #7221, regarding how to adjust cell output width in order to avoid that table columns are being wrapped to the next line past 80 characters. The following solution was posted by @joyceerhl in https://github.com/microsoft/vscode-jupyter/issues/7221#issuecomment-903891697_ :
> It sounds like you want to ensure that tables with a large number of columns don't get broken onto a new row? Does configuring `pd.options.display.max_columns` achieve what you want?  

Similar to this, I am trying to prevent columns from wrapping in my output in the Jupyter extension. However, I am running an R kernel, and I can't find a matching setting for Jupyter, R, or Markdown (`pd.` is a Pandas setting I believe), however, I do find similar settings for text editor: `diffEditor.wordWrap`, `editor.wordWrap`, and `editor.wordWrapColumn`. I tried to increase wrapping column size from 80 to 120, and, subsequently, set Word Wrap to `off` for Diff Editor and Editor, without any luck. I can only assume Text Editor settings do not apply to Jupyter notebooks? If it is supposed to apply to notebooks, the setting seem not to be working according to its intention. If not, is there a way I can turn off or increase wrapping limit for a Jupyter notebook running an R kernel?

Version info:
-   VS Code version: *1.66.2*
-   Jupyter Extension version: *v2022.3.1000901801*
-   Python Extension version: *v2022.4.1*
-   R Extension version: *v2.4.0*
-   OS and version: *Ubuntu 20.04.4 LTS*
-   Python and/or Anaconda version: *conda 4.11.0*
-   R version: *4.1.3*
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): *Not sure*
-   Jupyter server running: *Local*

**Comment by DonJayamanne** [comment]
this might help https://stackoverflow.com/questions/41271793/how-do-i-increase-the-number-of-columns-displayed-in-jupyter-with-r

**Reply by pehkawn** [reply]
Thanks for the tip! The link you posted addresses another nuisance I've been having with printing data frames and matrices. However, this relates to truncation of columns which is different from my current issue, where tables are wrapped, and columns that exceed a certain character limit are being wrapped and printed below as described in  #7221. For example, this happens when I try to print summary statistics from a model I've been running (using `summary()`). In this case the table of variables, estimates, etc., gets split at ~80 characters, and the columns that exceed this limit gets printed below.

**Comment by DonJayamanne** [comment, marked as answer]
From here https://github.com/microsoft/vscode-jupyter/issues/9808#issuecomment-1111544661

Yes! It worked! I went to Stack Overflow myself once I realised it's an R setting not an issue with Jupyter. I found a [similar problem/answer to your link](https://stackoverflow.com/q/12176296/9434803).
